automl/cloud-client/language_sentiment_analysis_create_model_test.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud import automl
+import pytest
+
+import language_sentiment_analysis_create_model
+
+PROJECT_ID = os.environ["GCLOUD_PROJECT"]
+DATASET_ID = "TST3960250460385409610"
```

Comment:
If this is a resource we expect the user to have created before hand, it should be in an environment variable (and trigger an error if it isn't set when the tests are run)